### PR TITLE
Zoom

### PR DESCRIPTION
**Pull request by ioandr**

### CODE REVIEW

**Comment by ioandr** [comment]
closes #14

File: src/line/line.js [hunk 1]
```diff
@@ -274,13 +274,27 @@ class LineGraph extends Graph {
...
+      // totalLength = d3.select(`.${classElement}`).select('.line').node().getTotalLength();
+      //
+      // // Animate line from left to right
+      // d3.select(`.${classElement}`).select('.line')
+      //   .attr('stroke-dasharray', `${totalLength} ${totalLength}`)
+      //   .attr('stroke-dashoffset', totalLength)
+      //   .transition()
+      //   .duration(1200)
+      //   .ease(d3.easeLinear)
+      //   .attr('stroke-dashoffset', 0);
```
Comment (drjova) [inline src/line/line.js]: can we remove that? :)

File: src/line/line.js [hunk 2]
```diff
@@ -489,16 +502,37 @@ class LineGraph extends Graph {
...
+        .duration(75)
+        .style('stroke-opacity', 1e-6)
+        .transition()
+        .duration(75)
         .call(that.makeGridlinesX(altX))
         .style('stroke-opacity', 0.7);
+
+      // Update animation
+      // totalLength = d3.select(`.${classElement}`).select('.line')
+      //   .node().getTotalLength();
```
Comment (drjova) [inline src/line/line.js]: can we remove that? :)

**Comment by drjova** [comment]
cherry-picked by a6e7d7408fce26019812c218c7d810f344da007c

**Pull request closed without merging**